src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,86 @@
+package duke.storage;
+
+import duke.DukeException;
+import duke.tasks.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Stores the user data in a text file.
+ */
+public class Storage {
+    private final File directory = new File("data");
+    private final File file = new File("data/tasks.txt");
```

Comment:
Would it be better to name these in ALL_CAPS and make them static variables since they appear to be constants?